docs/golden-path/plugins/backend/002-poking-around.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 002-poking-around
+sidebar_label: 002 - Poking around
+title: 002 - Poking around
+---
+
+## Default plugin functionality
+
+By default, that plugin that you just created hosts a simple todo list application. It exposes an HTTP API at `http://localhost:7007/api/todo/todos` that allows you to create TODOs, list existing TODOs, and get a specific TODO. It stores those TODOs in memory, which means that you would lose all of your TODOs if you restarted your application. It also allows you to tag TODOs with a Software Catalog entity, which will be useful for our frontend integration.
+
+To make this plugin production ready, we'll need to adjust a few things,
+
+1. Write our TODOs to a database so they don't get lost on restart.
+2. Write some proper tests to make sure everything works the way we expect.
+3. Get user feedback.
+
+## Testing locally
+
+Before we jump in to making this plugin ready to ship, let's walk through how to run it locally. If you open your backend plugin's manifest (`plugins/todo-backend/package.json`), and look at the `scripts` section, you'll notice a few important commands. The ones relevant to use right now are
+
+1. `yarn start` - Starts a local development server using the content in `dev/index.ts` as the backend.
+2. `yarn test` - Runs all of the tests for your backend plugin.
```

Comment:
It would be nice to have a short section on testing IMHO like using the `mockServices` etc ... I can't recall but I think the template this spins up contains test files as well? 💭  maybe we could dive into that a bit later on, just a thought 😄

(not saying it should be done in this PR btw!)